src/rules/flake8_django/rules.rs

```diff
@@ -0,0 +1,225 @@
+use crate::ast::types::Range;
+use crate::define_violation;
+use crate::registry::Diagnostic;
+use crate::violation::Violation;
+use ruff_macros::derive_message_formats;
+use rustpython_ast::Constant::Bool;
+use rustpython_ast::ExprKind::{Attribute, Call, Constant, Name};
+use rustpython_ast::StmtKind::{Assign, ClassDef, FunctionDef};
+use rustpython_ast::{Expr, Located, Stmt};
+
+define_violation!(
+    pub struct ModelDunderStr;
+);
+impl Violation for ModelDunderStr {
+    #[derive_message_formats]
+    fn message(&self) -> String {
+        format!("Model does not define __str__ method")
+    }
+}
+
+impl ModelDunderStr {
+    pub fn check_dunder_str(
+        bases: &[Expr],
+        body: &[Stmt],
+        class_location: &Stmt,
+    ) -> Option<Diagnostic> {
+        if !ModelDunderStr::checker_applies(bases, body) {
+            return None;
+        }
+        if !ModelDunderStr::has_dunder_method(body) {
+            return Some(Diagnostic::new(
+                ModelDunderStr,
+                Range::from_located(class_location),
+            ));
+        }
+        None
+    }
+
+    fn has_dunder_method(body: &[Stmt]) -> bool {
+        body.iter().any(|val| match &val.node {
+            FunctionDef { name, .. } => {
+                if name == "__str__" {
+                    return true;
+                }
+                false
+            }
+            _ => false,
+        })
+    }
+
+    fn checker_applies(bases: &[Expr], body: &[Stmt]) -> bool {
+        for base in bases.iter() {
+            if ModelDunderStr::is_abstract(body) {
+                continue;
+            }
+            if is_model(base) {
+                return true;
+            }
+        }
+        false
+    }
+
+    fn is_abstract(body: &[Stmt]) -> bool {
```

Comment:
Can you include a comment or docstring to indicate an example of the Python code that this is trying to catch? E.g., I initially assumed this would be checking to see if class is an `abc.ABC` or similar.